### PR TITLE
feat: Mask_RCNN TensorFlow 2.x training support

### PR DESCRIPTION
**Pull request by javoweb**
### Done
Added training support for Tensorflow 2.x
Tested in TensorFlow 2.4.0 (both CPU and GPU) and Onepanel DL v0.17 images. Model trains without any problems.
Inference in CVAT is performed without problems for Automatic Annotations; no changes in CVAT are required.
### Todo
- [x] **Important:** Multiprocessing was disabled for dataset generator(Known issue not planned to be fixed by TensorFlow since tf.data will replace the Sequential library for datasets). Dataset generator needs to be refactored to tf.data to improve training latency.
- [x] Add hyperparameter support to config MaskRCNN (current only epochs can be changed, included in this PR: num_steps)
- [ ] Minor: This model still uses some compat.v1 functions(tf.where). It needs to be refactored in the future. It still uses graph mode—more work is needed for full support of eager execution.

### CODE REVIEW

File: workflows/maskrcnn-training/template.yaml [hunk 1]
```diff
@@ -99,7 +81,7 @@ templates:
       workingDir: /mnt/src
     sidecars:
       - name: tensorboard
-        image: tensorflow/tensorflow:2.3.0
+        image: tensorflow/tensorflow:2.4.0
```
Comment (rushtehrani) [inline workflows/maskrcnn-training/template.yaml:84]: Might as well use `onepanel/dl:0.17.0` here too since it will already be cached in this node.

Comment (javoweb) [reply workflows/maskrcnn-training/template.yaml:84]: Sure.

File: workflows/maskrcnn-training/template.yaml [hunk 2]
```diff
@@ -1,26 +1,20 @@
...
-      hint: Select the last fine-tune checkpoint for this model. It may take up to 5 minutes for a recent checkpoint show here. Leave empty if this is the first time you're training this model.
+      hint: Path to the last fine-tune checkpoint for this model in default object storage. Leave empty if this is the first time you're training this model.
       displayName: Checkpoint path
       visibility: public
 
     - name: cvat-num-classes
       displayName: Number of classes
-      hint: Number of classes + 1 for background. In CVAT, this parameter will be pre-populated.
+      hint: Number of classes. In CVAT, this parameter will be pre-populated.
       value: '11'
```
Comment (rushtehrani) [inline workflows/maskrcnn-training/template.yaml:18]: Minor, but can you set this to an even field like `10`? That way they don't assume +1 for background

Comment (javoweb) [reply workflows/maskrcnn-training/template.yaml]: Done! :+1:

**Pull request closed without merging**